setup.cfg

```diff
@@ -1,2 +1,24 @@
 [bdist_wheel]
-universal = 1
\ No newline at end of file
+universal = 1
+
+[mypy]
+allow_redefinition = true
+check_untyped_defs = true
+pretty = true
+show_error_codes = true
+show_error_context = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+warn_unreachable = true
```

Comment:
warn in case mypy thinks this case can never happen (in case it actually can and the type checking is not checking a path that should be checked)